docs/about-lightning.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Lightning (Coming soon)
+sidebar_label: Lightning
+---
+
+## The Lightning pitch
+
+##### Leveraging the tech powering the field-tested enterprise OpenFn platform...
+
+Lightning brings together the tried and tested technology we have been using
+since 2015 (OpenFn core, engine, adaptors and code from the OpenFn SaaS
+platform) to manage the orchestration and execution of integrations in a stable,
+scalable and secure way.
+
+##### ...but providing a fully open source web app with a user-friendly web interface.
+
+A fully open source web app, it can be deployed anywhere through Docker and
+comes with a user-friendly, low-code interface with the full functionality
+needed for organizations to build, run and audit their workflows all in one
+place. ![Lightning](/img/lightning_build_run_audit.png)
+
+##### Build
+
+Empower more users in your organization to have a say in what gets automated and
+how. Lightning’s visual interface makes workflows more intelligible to
+non-technical users, bridging the gap between the IT specialists that build out
+automations and program managers that are the real business/ program experts on
+the processes that need automating.
+
+##### Audit
+
+Treat every workflow run with the care and attention it deserves. In OpenFn,
+each incoming request or transaction that gets processed is more than a piece of
+data - it represents a vulnerable child in need of critical support, a farmer
+managing their savings to make sure they can afford the next harvest. Lightning
+provides users with a dashboard that allows them to monitor the health of their
+integrations to make sure no request goes unprocessed.
+
+## What is Lightning ?
+
+[OpenFn/Lightning](https://github.com/OpenFn/lightning/) is a _fully open
+source_ workflow automation platform. It is designed for governments or NGOs who
+need a flexible solution which enables them to integrate and connect _any
+system_ and comes with fully-fledged user management and auditing capabilities.
+Lightning relies on the same tried-and-trusted core technology as the OpenFn
+iPaaS and comes with an improved, visual interface for building integrations.
+
+:::note
+
+Any integration built on the OpenFn iPaaS is can be run on Lightning.
+
+:::
+
+## Guiding principles
+
+Lightning is developed in line with the
+[principles for digital development](https://digitalprinciples.org/principles/)
+and under the guidance of it's Open Source Steering Committee which you can read
+about [here](https://openfn.github.io/governance/OSSC.html).
+
+On top of this, Lightning follows 4 key principles which determine how it should
+be developed:
+
+### Integrations break
```

Comment:
Maybe instead of having a header `Integrations break` (a line I don't want associated with Lightning), what about something like `Interoperability is an ongoing process` instead of the current title?

Reply:
Happy with this for now but would be good to think of something a bit shorter/more memorable down the line